Sources/Sentry/Public/SentryOptions.h

```diff
@@ -120,6 +120,11 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableOutOfMemoryTracking;
 
+/**
+ * Whether to enable measuring app start or not. Default is NO.
```

Comment:
L: should this be opt-in by default? cc @bruno-garcia

Reply:
I would rather disable it because we don't have any automatic transactions yet. This would then add overhead for no reason.

Reply:
mmm, not sure, so now people would need to enable performance tracking + this flag.
if its enabled by default (with enabled performance), they just need to capture a transaction with custom instrumentation, so one step more.